pywr-cli/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "pywr-cli"
+version = "0.1.0"
```

Comment:
Should the version across the separate crates be kept in sync, like egui? Not sure I like the approach that much but might make the release process easier?

Reply:
Good point. I don't think we'd necessarily need to keep them in-sync. We might just have to figure it out once we start releasing.